enterprise/models.py

```diff
@@ -94,21 +109,33 @@ class EnterpriseCustomerUser(TimeStampedModel):
     """
 
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
-    user_id = models.PositiveIntegerField(null=False, blank=False)
```

Comment:
@mtyaka That's a good point - I'll have to think about it a bit.

The problem here is that there's a use case when previously not seen email is sent. Such email should be recorded and when user registers with the email it will be automatically linked to EC. Current approach allows supporting this case (and other cases) without any hooks into registration workflow.

So, looks like we've got two options: 
1. Revert to use user_id and install some hook into registration workflow (so that when use uses an email already registered with EnterpriseCustomer EnterpriseCustomerUser record is updated to contain user_id)
2. Continue useing `user_email` - in this case we'll have to track email changes.

As I've said, I'll have to think about it, but suggestions welcome.